pkg/build/strategies/sti/sti.go

```diff
@@ -317,7 +317,6 @@ func (builder *STI) createBuildEnvironment() []string {
 	env, err := scripts.GetEnvironment(builder.config)
 	if err != nil {
 		glog.V(1).Infof("No user environment provided (%v)", err)
-		return nil
```

Comment:
The function createBuildEnvironment() was created by taking code that was duplicated, with the return nil added, I'm assuming by mistake. So my change just returns it into the previous state.